src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs

```diff
@@ -276,6 +285,19 @@ private static bool IsFileAlreadyOpen(string filePath)
             return false;
         }
 
+        private string GetCacheFilePath(string cacheKey)
```

Comment:
I only see that we are writing the cache file and deleting the cache file but we never read it. This tells me that there is a copy of this logic somewhere where we read the cached file. If this is the case the we should just one function that calculate this path.